dvc/ignore.py

```diff
@@ -216,7 +216,7 @@ def _parents_exist(self, path):
                 [os.path.abspath(path), self.tree_root]
             )
         ):
-            return False
+            return True
```

Comment:
This works as a temporary solution, but it still seems unintuitive.

It seems to me that dvcignore should only ever apply to a DVC repo, and by definition anything outside the repo root directory should not exist in the clean tree.